src/components/PrimaryLanguage.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+	<div class="wikibase-termbox wikibase-termbox--primaryLanguage">
+		<div>
+			<Dab :modifier="modifier" :language="primaryLanguage"/>
+		</div>
+		<div class="wikibase-termbox__actions">
```

Comment:
`.wikibase-termbox__actions` style moved into `Dab.vue` so got detached. As mentioned above, I think we should move the actions (once) into `TermBox.vue` (with the style obviously).